Flow.Launcher/MainWindow.xaml.cs

```diff
@@ -63,13 +63,78 @@ public MainWindow(Settings settings, MainViewModel mainVM)
             InitSoundEffects();
 
             DataObject.AddPastingHandler(QueryTextBox, OnPaste);
+
+            this.Loaded += (obj, args) =>
+            {
+                var handle = new WindowInteropHelper(this).Handle;
+                var win = HwndSource.FromHwnd(handle);
+                win.AddHook(new HwndSourceHook(WndProc));
+            };
         }
 
+        DispatcherTimer timer = new DispatcherTimer
+        {
+            Interval = new TimeSpan(0, 0, 0, 0, 500),
+            IsEnabled = false
+        };
```

Comment:
This timer seems to not be used anywhere?

Reply:
? I don't know what it is.